HPXMLtoOpenStudio/measure.rb

```diff
@@ -4272,8 +4272,33 @@ def self.add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces
       otherside_object.setName(exterior_adjacent_to)
       otherside_object.setCombinedConvectiveRadiativeFilmCoefficient(30)
       otherside_object.setMinimumOtherSideTemperatureLimit(temp_min)
-      otherside_object.setZoneAirTemperatureCoefficient(indoor_weight)
+      otherside_object.setConstantTemperatureCoefficient(indoor_weight)
       otherside_object.setExternalDryBulbTemperatureCoefficient(outdoor_weight)
+
+      if @other_side_indoor_sch.nil?
```

Comment:
All other side coefficient objects (no matter if of the same space type), share the same EMS program to read conditioned zone air temperature. 
EMS only reads zone temperature, not calculating other side temperature, letting othersidecoefficient object weigh between constant schedule value and outdoor air temperature. In this way, we can avoid duplicated EMS programs.